src/Schema.php

```diff
@@ -17,7 +17,7 @@ class Schema
     /**
      * The name of the endpoint
      *
-     * @var string
+     * @var \Muffin\Webservice\Model\Endpoint
```

Comment:
This doesn't seem right. There's `$this->_repository = $endpoint;` inside the class constructor and `$endpoint` is a string, so `$this->_repository` is also a string.

Reply:
Perhaps I've made a mistake in my test case. I will check and push any required updates.

Reply:
Yes, this is a mistake on my part. This change should have been in my Schema testing branch. I will amend.